src/System.Management.Automation/utils/Verbs.cs

```diff
@@ -592,6 +592,542 @@ public static class VerbsOther
     }
 
     /// <summary>
+    /// Verb descriptions.
+    /// </summary>
+    internal static class VerbDescriptions
+    {
+        /// <summary>
+        /// Gets verb description from the resource file
+        /// </summary>
+        public static string Get(string verb)
+        {
+            return VerbDescriptionStrings.ResourceManager.GetString(verb + "VerbDescription");
+        }
+    }//VerbDescriptions
+
+    /// <summary>
+    /// Verb Alias prefixes.
+    /// </summary>
+    internal static class VerbAliasPrefixes
```

Comment:
Maybe these should be attributes, like:

```c#
    public static class VerbsCommon
    {
        /// <summary>
        /// Synonyms: Add to, append or attach.
        /// </summary>
        [AliasPrefix("a")]
        public const string Add = "Add";
}
```

Reply:
Looks good as common pattern.
Should we do this in the _special_ case?